### DISCUSSION TITLE
Atom x5 z8350 Wifi not available & no sound

### DISCUSSION THREAD

**Discussion by plazon3** [sebanc/brunch#1339] [Q&A]
Hello,

I just installed brunch 94 on my PC stick with Atom x5 z8350 but there is no wifi available.
Also I can see the hdmi sound showing my inbuilt screen sound nzme, but i cant hear nothing.

could you help me on telling me which driver should i enable for the wifi to work and also which optin makes the sound work please?
im attaching the dmsg result.

thank you for your great work
[dmesg.txt](https://github.com/sebanc/brunch/files/7678260/dmesg.txt)

**Comment by tejasraman** [comment]
Run this command and tell me the output.

To run, press Ctrl+Alt+T, then type “shell” and type the following:
`sudo lshw -C network `

**Reply by tejasraman** [reply]
If you can boot into the OS…

**Comment by tejasraman** [comment]
Seems like there was no HDAudio controller detected (?) and maybe a driver wasn’t loaded. 

Try this too:
`sudo lshw -C sound`

**Reply by kiplik-ind** [reply]
says command not found, my laptop ryzen 5 with brunch n recovery zork 102 no sound too
plz advice me to do fix the sound.thanks alot

**Reply by tejasraman** [reply]
Try this:`lspci -v | grep Audio`

**Reply by tejasraman** [reply]
On _both_ of them. Intel cherry trail devices are always tricky to get audio on, but Ryzen should work out of the box. 

_PS - cherry trail is **old**, and there's still no OOTB audio for some of them. Curse Intel for this._

**Reply by kiplik-ind** [reply]
the result:
02:00.1 Audio device: Advanced Micro Devices, Inc. [AMD/ATI] Raven/Raven2/Fenghuang HDMI/DP Audio Controller
        Subsystem: Advanced Micro Devices, Inc. [AMD/ATI] Raven/Raven2/Fenghuang HDMI/DP Audio Controller
02:00.5 Multimedia controller: Advanced Micro Devices, Inc. [AMD] Raven/Raven2/FireFlight/Renoir Audio Processor
        Subsystem: Advanced Micro Devices, Inc. [AMD] Raven/Raven2/FireFlight/Renoir Audio Processor
02:00.6 Audio device: Advanced Micro Devices, Inc. [AMD] Family 17h (Models 10h-1fh) HD Audio Controller
        DeviceName: HD Audio Controller
still no sound

**Reply by tejasraman** [reply]
Use kernel 5.15 on the amd laptop and report back.

**Reply by kiplik-ind** [reply]
how to change kernel?

**Reply by kiplik-ind** [reply]
how to chenge kernel in chrome, i try config but no kernel chenged

**Reply by tejasraman** [reply]
Sorry for being late. To change, open the shell and type `sudo edit-brunch-config` and select 5.15. Don't change anything else ok?

**Comment by tejasraman** [comment]
Type sudo edit-brunch-config in the shell, and choose 5.15 when asked.

**Reply by kiplik-ind** [reply]
which command to view the kernel used on chromeos?

**Comment by kiplik-ind** [comment]
i allready to that, but still no sound
how to view kernel version used last revised/edited?

Pada tanggal Sab, 2 Jul 2022 pukul 09.03 tejasraman <
***@***.***> menulis:

**Comment by kiplik-ind** [comment]
chronos@localhost / $ uname -r
5.15.45-brunch-sebanc

**Comment by kiplik-ind** [comment]
cek result:
chronos@localhost / $ uname -a
Linux localhost 5.15.45-brunch-sebanc #1 SMP PREEMPT Sat Jun 25 14:43:08 UTC 2022 x86_64 AMD Ryzen 5 3500U with Radeon Vega Mobile Gfx AuthenticAMD GNU/Linux
still no suond

**Comment by tejasraman** [comment]
Weird... Try setting the `disable_intel_hda` option in edit-brunch-config as well as 5.15 and report back. (at least on the amd)

**Comment by kiplik-ind** [comment]
still no sound

**Comment by kiplik-ind** [comment]
is it coz conflict with USB web camera mic?
or sound driver not loaded on os ?

**Comment by tejasraman** [comment]
It could be that the Intel HD audio driver is conflicting with the AMD Ryzen 4000 (Renoir series) audio controller

**Comment by kiplik-ind** [comment]
already move to fyde os, n work well my laptop audio

Pada tanggal Jum, 1 Jul 2022 pukul 19.47 tejasraman <
***@***.***> menulis:
